optimade/server/routers/utils.py

```diff
@@ -56,7 +56,7 @@ def meta_values(
     return ResponseMeta(
         query=ResponseMetaQuery(representation=f"{url_path}?{parse_result.query}"),
         api_version=f"v{__api_version__}",
-        time_stamp=datetime.utcnow(),
+        time_stamp=datetime.now(),
```

Comment:
```suggestion
        time_stamp=datetime.utcnow(),
```
I guess there is no need to needlessly change this? Or?
I think keeping the `utc` here also makes it more clear that we're using UTC time.

Reply:
Without this change, your response will contain a completely wrong value (except when your local timezone is UTC). (please try it and let me know if I wrong I do not have access to any deployment). 
- Usually, a datetime object in python might contains timezone information as well. You can use any timezone that you wish so you need to use `astimezone`. So if you want `utcnow` your time value will be shifted twice.
- In general, the usage of `utcnow` is not recommended, because it does not represent the proper value of time. Please see the warnings on the official pythons documentation: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

Reply:
Right, so using `now()` will return a timezone-aware datetime? From the documentation, I don't think this is the case. Indeed it will also return a "naive" datetime object, since `tzinfo=None`.
So would it make sense to the add some timezone awareness for this line instead, e.g., `datetime.now(timezone.utc))`?

Reply:
The answer is no and I agree it is not the case! Python handles the "native" datetime object as local time (please see the link above).  Of course, you can use `datetime.now(timezone.utc))` because it is equivalent to `datetime.now()`. Please see the following code below:
```python
from datetime import datetime, timezone

print("Local time in local time zone (default in python)")
time_now = datetime.now()

print(time_now)
print(time_now.isoformat())
print(time_now.astimezone(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ"))

print("Local time in UTC time zone")
time_now_utc = datetime.now(timezone.utc)

print(time_now_utc)
print(time_now_utc.isoformat())
print(time_now_utc.astimezone(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ"))

print("The UTC time in local time zone (WRONG!)")
time_utcnow = datetime.utcnow()

print(time_utcnow)
print(time_utcnow.isoformat())
print(time_utcnow.astimezone(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ"))
```
```Local time in local time zone (default in python)
2020-06-15 15:13:43.974074
2020-06-15T15:13:43.974074
2020-06-15T13:13:43Z
Local time in UTC time zone
2020-06-15 13:13:43.974151+00:00
2020-06-15T13:13:43.974151+00:00
2020-06-15T13:13:43Z
The UTC time in local time zone (WRONG!)
2020-06-15 13:13:43.974183
2020-06-15T13:13:43.974183
2020-06-15T11:13:43Z
```
You can choose to define the time as local with not timezone (default in python) or you can define a different value at different timezone but doesn't make sense to define the UTC value as a local time zone...

If it is confusing as an alternative way you can convert any datetime object to string in-place and just use it as a string.

Reply:
This is quite confusing, but I'm glad you're here to spell it out very clearly. Thanks!